node/impl/paych/paych.go

```diff
@@ -160,6 +165,9 @@ func (a *PaychAPI) paychVoucherCreate(ctx context.Context, pch address.Address,
 	sv.Signature = sig
 
 	if _, err := a.PaychMgr.AddVoucherOutbound(ctx, pch, sv, nil, types.NewInt(0)); err != nil {
+		if err == paychmgr.ErrInsufficientFunds {
```

Comment:
I don't think there's currently a way to return different error codes through the API. So my proposal here is to return nil if there are not enough funds in the channel.